<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>756</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Организация научных выставок</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>source/database-icon.png</normaloff>source/database-icon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="1" column="0" colspan="4">
      <widget class="QTableView" name="tableView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="recNum_lb">
       <property name="text">
        <string>Всего строк: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="2" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="CreateNewTable_bn">
         <property name="text">
          <string>Создать таблицу</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="deleteTable_bn">
         <property name="text">
          <string>Удалить группу НИР</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="editEntery_bn">
         <property name="text">
          <string>Добавить запись в таблицу</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EditField">
         <property name="text">
          <string>Редактировать запись</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="AddField">
         <property name="text">
          <string>Добавить запись</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Удалить запись</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="dbInfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="filtering_bn">
       <property name="text">
        <string>Фильтрация</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="ChoiceTable_menu">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="action_sourceTable"/>
    <addaction name="action_researchTable"/>
   </widget>
   <widget class="QMenu" name="menu_eq">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <addaction name="ChoiceTable_menu"/>
   <addaction name="menu_eq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnf_kbwf">
   <property name="text">
    <string>nf,kbwf</string>
   </property>
  </action>
  <action name="action1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Таблица</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>ТАблица</string>
   </property>
  </action>
  <action name="action_sourceTable">
   <property name="icon">
    <iconset>
     <normaloff>source/data-integration-icon.png</normaloff>source/data-integration-icon.png</iconset>
   </property>
   <property name="text">
    <string>Исходные данные</string>
   </property>
  </action>
  <action name="action_researchTable">
   <property name="icon">
    <iconset>
     <normaloff>source/create-table-icon.png</normaloff>source/create-table-icon.png</iconset>
   </property>
   <property name="text">
    <string>Группы НИР</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
